lib/firecord/repository/response.rb

```diff
@@ -0,0 +1,27 @@
+module Firecord
+  module Repository
+    class Response
+      def initialize(response, default = {})
+        @response = response
+        @default = default
+      end
+
+      def sanitize(expand_error = true)
```

Comment:
Firecord::Repository::Response#sanitize has boolean parameter 'expand_error'

[Read more about it here.](https://github.com/troessner/reek/blob/master/docs/Boolean-Parameter.md)